qa/tasks/fwd_scrub.py

```diff
@@ -35,7 +35,7 @@ def _run(self):
             self.do_scrub()
         except Exception as e:
             self.set_thrasher_exception(e)
-            self.logger.exception("exception:")
+            self.logger.error(f"exception: {e}")
```

Comment:
Well, also be aware that `tasks.thrasher.ThrasherGreenlet.Stopped` is not even an error. The `_run` method should simply return when stopped.

Reply:
If that's the case then it makes no sense to raise `self.Stopped()` here which is done when `raise_stopped` value is `True` which is by default true. We can make it `False` by default then. I guess it would suffice our requirement.

Reply:
@batrick ping

Reply:
No, I think the fix is:

```
+except ThrasherGreenlet.Stopped as e:
+    pass
except Exception as e:
    ....
```

no?

Reply:
> No, I think the fix is:
> 
> ```
> +except ThrasherGreenlet.Stopped as e:
> +    pass
> except Exception as e:
>     ....
> ```
> 
> no?

This is fine,  just passing while encountering this exception. Not sure, but while reading the code I thought if don't need this exception, we can simply don't raise it by setting `raise_stopped` to false by default. Anyways,if its not correct, I will update with the suggested fix.

Reply:
The point of this ticket is to avoid printing anything out for an expected non-error exception. So I think it is the right one.